featuretools/tests/primitive_tests/transform_primitive_tests/test_savgol_filter.py

```diff
@@ -0,0 +1,266 @@
+            22.478260869564927,
+            23.15233736515171,
+            24.12356979405003,
+            25.23962079110788,
+            26.000980712650854,
+            27.082379862699877,
+            27.787839163124843,
+            28.879045439685797,
+            29.762994442627924,
+            31.067342268714864,
+            32.11147433801854,
+            32.666557698593884,
+            33.06864988558309,
+            34.00098071265075,
+            35.134030728995945,
+            36.15135665250035,
+            36.945733899966825,
+            37.56227525335028,
+            38.55769859431137,
+            39.3975155279498,
+            39.87054593004198,
+            40.304347826086435,
+            41.11670480549146,
+            42.00948022229432,
+            41.982674076495044,
+            42.62798300098016,
+            43.15887544949274,
+            44.53481529911678,
+            45.680614579927486,
+            46.93886891140834,
+            47.98300098071202,
+            48.80549199084604,
+            50.28244524354299,
+            52.66851912389601,
+            54.28604118993064,
+            55.81529911735788,
+            57.10297482837455,
+            57.82641386073805,
+            59.45276234063342,
+            60.77280156913945,
+            61.23667865315383,
+            61.81660673422607,
+            62.60281137626594,
+            62.54004576658957,
+            62.78653154625613,
+            63.23046747302958,
+            64.09087937234307,
+            65.25661981039471,
+            65.19385420071833,
+            66.34161490683144,
+            66.65021248774022,
+            67.38280483818154,
+            68.8126838836212,
+            69.79470415168265,
+            70.943772474664,
+            72.74076495586698,
+            73.04020921869797,
+            73.3586139261187,
+            74.67734553775647,
+            75.71559333115299,
+            77.51814318404607,
+            79.62471395880902,
+            80.60150375939745,
+            80.61163779012645,
+            81.89342922523593,
+            82.41124550506593,
+            83.19293292519846,
+            83.97174920172642,
+            84.7620599588564,
+            85.57823082079385,
+            86.4346274117442,
+            87.34561535591293,
+            88.32556027750543,
+            89.38882780072717,
+            90.54978354978357,
+            91.82279314888011,
+        ],
+    )
+
+    def test_error(self):
+        window_length = 1
+        polyorder = 3
+        error_text = "polyorder must be less than window_length."
+        with raises(ValueError, match=error_text):
+            self.primitive(window_length, polyorder)
+
+        error_text = (
+            "Both window_length and polyorder must be defined if you define one."
+        )
+
+        with raises(ValueError, match=error_text):
+            self.primitive(window_length=window_length)
+            self.primitive(polyorder=polyorder)
```

Comment:
This second line maybe needs to be in its own 'with' block?

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/2485/commits/f99f4c1a84be8a84630b8fc7fa7c0e75c25e3967